app/javascript/react/screens/App/Plan/PlanConstants.js

```diff
@@ -29,6 +29,7 @@ export const STATUS_MESSAGE_KEYS = {
   POWER_OFF: 'Power off',
   POWER_ON: 'Power-on VM',
   PRE_MIGRATION: 'Pre-migration',
+  PRE_MIGRATE_VM: 'Pre Transform VM',
```

Comment:
Do we want the value to be `Pre Migrate VM`, or do we want our constants to be `PRE_TRANSFORM_VM`?

Reply:
down below we have a `MIGRATE_VM: 'Transform VM',` -- so just wanted to be consistent with that.
But I'd be OK to change both these entries to `TRANSFORM`
LMK if I should.

Reply:
Ah yes, I see that now.  Let's change to `TRANSFORM`, if it's not too much trouble (not sure how many places we'd need to change that).  It's confusing to me because I can't figure out which part of the dictionary entry is "correct".  Plus it's consistent with the rest of the key/values :)

Reply:
Ok, not a problem.

I have changed all MIGRATE/MIGRATION entries to TRANSFORM/TRANSFORMATION in `STATUS_MESSAGE_KEYS`